minNumber/main.c

```diff
@@ -0,0 +1,76 @@
+﻿#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define OK 0
+#define BAD_INPUT 1
+#define TESTS_FAILED 2
+
+#define MAX_DIGITS_NUMBER 10
+
+int compareCharacters(const char* const a, const char* const b)
+{
+    return (int)*a - (int)*b;
+}
+
+int getMinNumber(char* const stringNumber)
```

Comment:
```suggestion
int getMinNumber(const char* const stringNumber)
```
Иначе ниже при вызове функции, где сюда константная строка передаётся, предупреждение, что константную строку могут поменять.